articles/cosmos-db/restore-account-continuous-backup.md

```diff
@@ -424,7 +424,7 @@ az cosmosdb restore \
 
 ```
 
-If `**--enable-public-network**` is not set, restored account is accessible from public network. Please ensure to pass `` False` to the `**--enable-public-network** `` option to prevent public network access for restored account.
+If `--enable-public-network` is not set, restored account is accessible from public network. Please ensure to pass `False` to the `--enable-public-network` option to prevent public network access for restored account.
```

Comment:
If the intent here is to make this bold shouldn't the "**" just precede the tilde "`"?